nasa/harmony
# DAS-1766 - Do not batch NetCDF-to-Zarr outputs

## Jira Issue ID

DAS-1766

## Description

This PR updates all NetCDF-to-Zarr services chains to prevent the output from being batched.

I also had to update some of the unit tests, as they were using one of the NetCDF-to-Zarr service chains to verify ignore-errors functionality with a batched service. I swapped those tests over to use a service chain of the PO.DAAC L2 Subsetter and Concise services.

## Local Test Steps

* Pull this branch.
* Restart your Harmony to use the updated configuration file.
* Perform the following request:
```
localhost:3000/C1245618475-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&format=application%2Fx-zarr&concatenate=true&maxResults=450
```

You should see that there is only one output Zarr store.

## PR Acceptance Checklist
* ~~Documentation updated (if needed)~~
